platform-includes/metrics/configure/javascript.nextjs.mdx

```diff
@@ -4,11 +4,20 @@ Metrics work out of the box by calling `Sentry.init` no further setup is require
 // sentry.client.config.js
 Sentry.init({
   dsn: '___PUBLIC_DSN___',
+  // Only needed for SDK versions < 8.0.0
+  // integrations: [
+  //   Sentry.metrics.metricsAggregatorIntegration(),
+  // ],
 });
 ```
 
 ```JavaScript
+// instrumentation.js
```

Comment:
We had not filename for the server side configuration snippet of NextJS.
From what I gathered it should be `instrumentation.js`?